WebApiCoreSeed.Data/Models/User.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace WebApiCoreSeed.Data.Models
+{
+    using System.ComponentModel.DataAnnotations;
+
+    /// <summary>
+    /// Represents a person in our system
+    /// </summary>
+    public class User : BaseEntity
+    {
+        /// <summary>
+        /// First name of the person.
+        /// </summary>
+        [Required]
+        public string FirstName { get; set; }
+
+        /// <summary>
+        /// Last name, family name or surname of the person.
+        /// </summary>
+        [Required]
+        public string LastName { get; set; }
+
+        /// <summary>
+        /// Email address of the person. This field is unique for each user.
```

Comment:
Since we think it might be possible to have two users with the same email address, we just updated the comment accordingly.